espnet/asr/pytorch_backend/asr.py

```diff
@@ -1116,6 +1116,13 @@ def recog(args):
                                 nbest_hyps[n]["score"] += hyps[n]["score"]
                 elif hasattr(model, "is_rnnt"):
                     nbest_hyps = model.recognize(feat, beam_search_transducer)
+                elif hasattr(model, "decoder_mode") and args.wfst_decode:
+                    grpha = torch.load('data/graph/HLG.pt')
```

Comment:
It's better to specify the file path of `HLG.pt` instead of this hard-coded path.